### PR TITLE
docs(migrate) mention internal changes in migration guide

### LINKED ISSUE #2393: Document webpack change: add some types
*A pull request by @sokra was merged and maintainers requested a documentation change.*

See pull request: https://github.com/webpack/webpack/pull/7786

---

**What kind of change does this PR introduce?**
types

**Did you add tests for your changes?**
N/A

**Does this PR introduce a breaking change?**
yes

**What needs to be documented once your changes are merged?**
Migration guide:
`Module.nameForCondition` `Module.updateCacheModule` and `Module.chunkCondition` are no longer optional. Module should extend `Module` and this is no problem.

### PR DESCRIPTION
**Pull request by EugeneHlushko**
- Closes #2393 
- mention that internals changes are listed just for information but not necessarily valuable for common users

### CODE REVIEW

File: src/content/migrate/5.md [hunk 1]
```diff
@@ -156,3 +156,8 @@ Create an [issue](https://github.com/webpack/webpack/issues/new?template=Bug_rep
 
 Please open a Pull Request to help the next person using this guide.
 
+## Changes to internals
+
+The changes to webpack internals like for example: adding types, refactoring code and methods renaming are listed here for anyone interested. But they are not intended as a part of common use-case migration.
```
Comment (anshumanv) [inline src/content/migrate/5.md]:
```suggestion
The changes to webpack internals such as: adding types, refactoring code and methods renaming are listed here for anyone interested. But they are not intended as a part of common use-case migration.
```

Comment (anshumanv) [reply src/content/migrate/5.md]: Or `like` can for too

Comment (EugeneHlushko) [reply src/content/migrate/5.md]: Do we need a comma befire such as? I am not sure what you guys thing?

Comment (anshumanv) [reply src/content/migrate/5.md]: like and for example seems a bit redundant to me together conveying the same meaning, we can use either of them or something like `such as`.

Comment (EugeneHlushko) [reply src/content/migrate/5.md]: yes i was saying if we might need to add a comma before `such as`

**Pull request merged**